src/gettingstarted/cli/api-tokens.md

```diff
@@ -19,28 +19,30 @@ After creating the token it will be displayed once at the top of the page in a g
 Now set that token to an environment variable named `PLATFORMSH_CLI_TOKEN` on your system where the CLI will run.  Consult the documentation for your CI system to see how to do that.
 
 > **note**
->
 > If running CLI commands from any automated system, including a Platform.sh cron task, we urge you to use the `--no-wait` flag on any commands that may take more than a second or two to avoid blocking the process.
 
-## Automation users
+## Machine users
 
-For projects with more than one user it is often a good idea to create a dedicated "automation user" to run automation tasks such as backups. Like human users, every user account needs its own unique email address.
- 
-The automation user can be given a very restrictive set of permissions limited to just its needed tasks. Backups, for instance, require `Admin` access but no SSH key, while checking out code from a CI server to run tests on it would require an SSH key but only `Reader` access.
- 
-It will also show up in logs and activity streams as a separate entry from human users. We recommend having a separate automation user for each project to be automated.
+For security reasons we recommend to create a dedicated `machine user` to run automation tasks such as taking backups, renewing SSL certificates or triggering source operations. We also recommend to create a unique `machine user` for each project to be automated. 
```

Comment:
Machine user is not a code string here, so it shouldn't be using code offsets.  Normal quotes are fine.